src/overlays/actors/ovl_En_Holl/z_en_holl.c

```diff
@@ -116,34 +144,67 @@ void EnHoll_SwapRooms(PlayState* play) {
     play->roomCtx.unk_30 ^= 1;
 }
 
-// Horizontal Planes
-void func_80A58DD4(EnHoll* this, PlayState* play) {
+/**
+ * When traversing a holl of this kind, it attempts to:
+ *   Load Current Room (fails as it is already loaded)
```

Comment:
not saying it needs to change now, but hopefully we come up with a better word than "holl" in the future lol

Reply:
"room transition region" or the familiar "loading zone" may work, but it would possibly encompass more than EnHoll :thinking:

Reply:
yea. loading zone, at least in my experience, more pertains to scene changing loading zones. its tough for sure.
but yeah, will probably be a relevant discussion when we come up with new actor names in 2032